### PR TITLE
Farfi local

### PR DESCRIPTION
**Pull request by Farfi**
How many languages people speak in Barcelona?

### CODE REVIEW

**Comment by ismaelcrespof** [comment]
I agree

File: Farfi-barcelona.md [hunk 1]
```diff
@@ -6,5 +6,7 @@
 
 ### Touristy things to do
 - Visit some monuments
+- Walk through Ramblas
```
Comment (ismaelcrespof) [inline Farfi-barcelona.md]: Hi there, this is a sample comment

File: Farfi-barcelona.md [hunk 2]
```diff
@@ -6,5 +6,7 @@
...
 
 ### Secret spots from locals
+- Bunker del Carmel
```
Comment (ismaelcrespof) [inline Farfi-barcelona.md]: Bunker? Really?

Comment (ismaelcrespof) [inline Farfi-barcelona.md]: Incredible!

File: Farfi-barcelona.md (hunk 1 above)
Comment (tonyjobson) [inline Farfi-barcelona.md]: How long should we schedule to walk through Ramblas and what is it?

Comment (ismaelcrespof) [inline Farfi-barcelona.md]: An hour is ok

**Pull request merged**